pkg/internal/apis/config/validate.go

```diff
@@ -168,6 +168,10 @@ func validateSubnets(subnetStr string, dualstack bool) error {
 	// if not DualStack only one CIDR allowed
 	case !dualstack && len(subnets) > 1:
 		allErrs = append(allErrs, errors.New("only one CIDR allowed for single-stack networking"))
+	case ipFamily == IPv4Family && subnets[0].IP.To4() == nil:
```

Comment:
can subnets be 0?
maybe we should be paranoid and consider that to avoid a panic here

Reply:
Fair enough, I added a case len == 0 on top of the switch